Acl/RoleCommentAcl.php

```diff
@@ -81,12 +78,12 @@ public function __construct(SecurityContextInterface $securityContext,
                                 $commentClass
     )
     {
-        $this->securityContext   = $securityContext;
-        $this->createRole        = $createRole;
-        $this->viewRole          = $viewRole;
-        $this->editRole          = $editRole;
-        $this->deleteRole        = $deleteRole;
-        $this->commentClass      = $commentClass;
+        $this->authorizationChecker = $authorizationChecker;
```

Comment:
please add a validation of ``$authorizationChecker`` at the beginning of the constructor (to replace the typehint as we cannot have a typehint accepting both APIs)